production/helm/loki/values.yaml

```diff
@@ -671,6 +671,13 @@ monitoring:
       enableConfigReadAPI: false
       # -- The name of the PriorityClass for GrafanaAgent pods
       priorityClassName: null
+      # -- Resource requests and limits for the grafanaAgent pods
+      resources: {}
+      #   limits:
+      #     memory: 200Mi
+      #   requests:
+      #     cpu: 50m
+      #     memory: 100Mi
       # -- Tolerations for GrafanaAgent pods
```

Comment:
I don't think this PR should be blocked on it but we should use Pods instead of pods for consistency with Kubernetes documentation.

I've opened https://github.com/grafana/loki/issues/11944.